dhall/src/Dhall/Eval.hs

```diff
@@ -1273,9 +1274,9 @@ alphaNormalize = goEnv EmptyNames
             None ->
                 None
             Record kts ->
-                Record (goRecordField <$> kts)
+                Record (Lens.over Syntax.recordFieldExprs go <$> kts)
             RecordLit kts ->
-                RecordLit (goRecordField <$> kts)
+                RecordLit (Lens.over Syntax.recordFieldExprs go <$> kts)
```

Comment:
Actually the previous version seems clearer. This version has more "non-essential" complexity that we're trying to avoid in the core modules.

 Could you revert efb4cfde2f766b3d8b5a0fffeb9af65116dec1a0?